paltas/Sources/sersic.py

```diff
@@ -315,8 +326,11 @@ def draw_source(self):
 			'n_sersic':self.source_parameters['n_disk'],
 			'e1':self.source_parameters['e1_disk'],
 			'e2':self.source_parameters['e2_disk']}
-		light_model_kwargs = [kwargs_bulge,kwargs_disk]
 
+		return self._complete_double_sersic_draw(mag_bulge, kwargs_bulge, mag_disk, kwargs_disk)
+
+	def _complete_double_sersic_draw(self, mag_bulge, kwargs_bulge, mag_disk, kwargs_disk):
```

Comment:
This change makes sense to me, but it would be good to write an explicit test for this function now.